src/Serilog.AspNetCore/SerilogWebHostBuilderExtensions.cs

```diff
@@ -41,5 +41,25 @@ public static IWebHostBuilder UseSerilog(this IWebHostBuilder builder, Serilog.I
                 collection.AddSingleton<ILoggerFactory>(new SerilogLoggerFactory(logger, dispose)));
             return builder;
         }
+
+        /// <summary>Sets Serilog as the logging provider.</summary>
```

Comment:
May need to include a note on how this differs from the overload above, e.g. `Sets Serilog as the logging provider. A <see cref="WebHostBuilderContext"/> is supplied so that configuration and hosting information can be used.`

Reply:
Also should possibly add `The logger will be shut down when application services are disposed.`